docs/client.md

```diff
@@ -14,7 +14,7 @@ The code to create a new client looks like:
 
     var ldap = require('ldapjs');
     var client = ldap.createClient({
-      url: 'ldap://127.0.0.1:1389'
+      url: 'ldap://127.0.0.1:1389,127.0.0.2:1389'
```

Comment:
IF we are going to support multiple servers, the url parameter should be an array.

Reply:
I can add support for arrays, but having single string as a connection string with multiple servers is quite usual.. some examples:
mongodb: https://github.com/mongodb/node-mongodb-native/blob/master/src/connection_string.ts#L139-L149
zookeeper: https://github.com/alexguan/node-zookeeper-client/blob/master/lib/ConnectionStringParser.js#L51-L55

Reply:
Because they do it doesn't mean it is a good idea. I would much prefer an array of URLs.